### PR TITLE
backports.test.support: add recipe

### PR DESCRIPTION
**Pull request by nehaljwani**

### CODE REVIEW

**Comment by mwcraig** [comment]
@nehaljwani -- thanks for already reporting (and fixing!)  the problem that needed a patch upstream. Would you mind updating the version of `backports.test.support` so that the patch is not needed?

**Comment by nehaljwani** [comment]
I have update the PR. Thank you for reviewing @mwcraig !

**Comment by mwcraig** [comment]
@nehaljwani -- could you please also delete the patch file from the recipe?

**Comment by nehaljwani** [comment]
Whoops. Fixed.

File: recipes/backports.os/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  requires:
+    - hypothesis
+  # Unicode + py27 on Windows -> :'(
+  commands:                              # [unix]
+    - export "PYTHONIOENCODING=utf8"     # [unix]
+    - python -m unittest discover tests  # [unix]
+
+about:
+  home: https://github.com/pjdelport/backports.os
+  license: Python Software Foundation
```
Comment (jakirkham) [inline recipes/backports.os/meta.yaml]: FYI we normally include a version with these (e.g. `PSF 2`).

File: recipes/backports.os/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - setuptools_scm
+  run:
+    - python
```
Comment (jakirkham) [inline recipes/backports.os/meta.yaml]: This should require `backports` too, correct?

**Comment by jakirkham** [comment]
Thanks for adding this. Very useful given recent Python 2.7 changes.

**Pull request merged**